### DISCUSSION TITLE
How to import a list of csv files from different paths

### DISCUSSION THREAD

**Discussion by djouallah** [ClickHouse/ClickHouse#69366] [Q&A] [answered]
I am trying to read a list of csv files from different directory, although they have the same schema basically

List_csv_file =['directory1/dddd/file_45899.csv', 'zzzz/dddd/fff/file_9uu.csv' etc] the paths have nothing in common

it will nice that clickhouse file function accept an arbitrary list of paths as it is supported by other engines , polars, duckdb etc

thanks and appreciate any workaround

**Comment by alexey-milovidov** [comment, marked as answer]
Write them as a set in curly braces: `{directory1/dddd/file_45899.csv,zzzz/dddd/fff/file_9uu.csv}`
